head/tests/CMakeLists.txt

```diff
@@ -28,6 +29,9 @@ target_compile_options(head
         -Wextra-semi
         -Wctor-dtor-privacy
         -fno-rtti)
+
+target_head_core_tool_list(head)
```

Comment:
Do we need this? can't we use `target_head_core`?

Reply:
I was trying to keep FreeRTOS out of tests at the time I think - 
```
In file included from /Users/aatirmanzur/code/ot3-firmware/head/tests/../../include/can/core/message_writer.hpp:8,
                 from /Users/aatirmanzur/code/ot3-firmware/head/tests/../../include/head/core/tasks.hpp:4,
                 from /Users/aatirmanzur/code/ot3-firmware/head/core/tasks.cpp:1:
/Users/aatirmanzur/code/ot3-firmware/head/tests/../../include/common/core/freertos_message_queue.hpp:5:10: fatal error: FreeRTOS.h: No such file or directory
    5 | #include "FreeRTOS.h"
      |          ^~~~~~~~~~~~
compilation terminated.
```